Pidaq/Lib/wind_tunnel/example.py

```diff
@@ -0,0 +1,25 @@
+import logging
+import os
+import sys
+from air_properties import LinearInterpolation
+
+logger = logging.getLogger()
+
+if __name__ == "__main__":
+    while 1:
+        filePath = input("CSV File Path, no quotations: ")
+        
+        # Exits the loop if the path is valid
+        if os.path.exists(filePath):
+            break
+        else:
+            logger.warning("Path does not exist. \nPlease enter a valid path: ")
+
+    interObject = LinearInterpolation(filePath)
+    value = input("Enter temperature value, or write \"exit\" to end: ")
+    while(value != "exit"):
+        x = interObject.interpolateDensity(value)
+        y = interObject.interpolateViscosity(value) 
+        print(x) 
+        print(y)
+        value = input("Enter next temperature value, or write \"exit\" to end: ")
```

Comment:
Although this code is mainly for testing, It may be a good idea to print what value is which (mostly to make this look cool hahaha)

Reply:
Agreed. Done, and it does look cool.